tests/unit/lib/test_schema.py

```diff
@@ -0,0 +1,22 @@
+from datachain.lib.dc import C, DataChain
+from datachain.lib.file import File
+
+
+def test_column_filter_by_regex():
```

Comment:
> Need to test if it works on CH the same way and regexp supported is more or less the same.

I think if you add the `catalog` fixture here you should be able to confirm that this works for CH.